spec/computer_player_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "spec_helper"
+require "tictactoe"
+
+RSpec.describe ComputerPlayer do
+  class ComputerPlayer::FakeBoard
```

Comment:
Maybe we could get a more specific name? I would be in favor of something along the lines of `BoardWithOnlyMiddleSpaceAvailable` 😄

In general, I think it would be good for us to get away from the `TestClass::FakeCollaborator` pattern. Using more specific names for test doubles will help the readability of the tests. A reader is also less like to need to ask the question how is `HumanPlayer::FakeBoard` different from `ComputerPlayer::FakeBoard`?

Reply:
Sounds good